.snippets/code/mrl/polkadotxcm-send.js

```diff
@@ -0,0 +1,55 @@
+const moonbeamProvider = new WsProvider(MOONBEAM_CHAIN_WS_ENDPOINT);
+const moonbeamAPI = await ApiPromise.create({ provider: moonbeamProvider });
+
+const ethereumTx = await batchApproveTransferTx(moonbeamAPI);
+const txWeight = (await ethereumTx.paymentInfo(MLD_ACCOUNT)).weight;
+
+const xcmExtrinsic = originChainPolkadotJsAPI.tx.polkadotXcm.send(
+  { V3: { parents: new BN(1), interior: { X1: { Parachain: 1000 } } } },
+  {
+    V3: [
+      // Withdraw DEV asset (0.06) from the target account
```

Comment:
why 0.06 DEV? does this number come from somewhere specific?

Reply:
No, it's a slight overestimation for what is required. It's just enough to pay for EVM tx gas and XCM fees